start.py

```diff
@@ -1,14 +1,26 @@
 #!/usr/bin/env python3
 from lib.exp import Client
+import json
+import os
 import uuid
 from kubernetes import client as k8sclient
 
 
+ns = os.getenv("EXPERIMENT_NAMESPACE")
+if not ns:
```

Comment:
I think you can skip this, if you provide "demo" as the second argument for getenv?